api/routes/assessment_routes.py

```diff
@@ -477,6 +483,33 @@ def update_flag_v2_for_application():
     return AssessmentFlagSchema().dump(updated_flag)
 
 
+def associate_tags_with_assessment(application_id):
+    args = request.get_json()
+    tags = args
+    current_app.logger.info(f"Associating tag with assessment")
+    associated_tags = associate_assessment_tags(application_id, tags)
+
+    if associated_tags:
+        serialiser = TagAssociationSchema()
+        serialised_associated_tags = [
+            serialiser.dump(r) for r in associated_tags
+        ]
+        return serialised_associated_tags
+
+
+def get_tags_associated_with_assessment(application_id):
+    current_app.logger.info(
+        f"Getting tags associated with assessment with application_id: {application_id}."
+    )
+    associated_tags = select_tags_associated_with_assessment(application_id)
+    if associated_tags:
+        serialiser = JoinedTagAssociationSchema()
+        serialised_associated_tags = [
+            serialiser.dump(r) for r in associated_tags
+        ]
+        return serialised_associated_tags
```

Comment:
Do we need the if here? isn't a valid use case to return an empty list? Rather than `None` - might be better for assessment consumption side.  I can refactor this as I'm working on the front-end tho if you agree